sencha-workspace/SlateDemonstrationsStudent/app/controller/MainController.js

```diff
@@ -4,8 +4,8 @@
  *
  * TODO - Replace this content of this view to suite the needs of your application.
  */
-Ext.define('SlateDemonstrationsStudent.view.main.MainController', {
-    extend: 'Ext.app.ViewController',
+Ext.define('SlateDemonstrationsStudent.controller.MainController', {
```

Comment:
@aliyeysides it's already in the controller namespace you don't need to call it controller again, you won't see `Controller` appended to any other global controllers. Just call this `controller.Main` or `controller.Dashboard`